geometry_msgs/Point pointL1
geometry_msgs/Point pointL2
